features/step_definitions/outcome.rb

```diff
@@ -73,4 +73,12 @@
 
 Alors('l\'année de publication est {string}') do |string|
   expect(find('.metadata > .edition', match: :first)).to have_content string
+end
+
+Alors('le texte à la fin du commentaire contient la nouvelle référence temporelle') do 
+  expect(find('.editable.content').text).to match(/[0-9:.]+ --> [0-9:.]+\n<TEXT>/)
```

Comment:
- You should test the *value* of the `textarea` instead.
- Test the regex to be sure:
  - it is multiline,
  - the dot match only the real dot and not any character.